images/router/haproxy/conf/haproxy-config.template

```diff
@@ -569,7 +606,7 @@ backend be_tcp:{{$cfgIdx}}
           "<cert>: <domain-set>" is important as this allows us to use
          wildcards and/or use a deny set with !<domain> in the future.
 */}}
-{{ define "/var/lib/haproxy/conf/cert_config.map" -}}
+{{ define "/var/lib/haproxy/conf/.tmp/cert_config.map" -}}
```

Comment:
Why do the temp file instead of just having a sort helper for .State?

Reply:
so what would make more sense is to just generate the sorted config info via different helper functions [its different config in some cases] and just do a simple `range` in the template and put that info into a file. There was also pushback on adding more helper functions - mentioned on the sprint calls and from the past, so ...